datalake/src/main/java/com/sequenceiq/datalake/service/sdx/StackRequestManifester.java

```diff
@@ -170,6 +171,9 @@ private StackV4Request setupStackRequestForCloudbreak(SdxCluster sdxCluster, Det
             setupMultiAz(sdxCluster, environment, stackRequest);
             setupGovCloud(sdxCluster, environment, stackRequest);
             stackRequest.setExternalDatabase(databaseRequestConverter.createExternalDbRequest(sdxCluster));
+            if (entitlementService.awsNativeSingleAzDatalakeEnabled(Crn.fromString(sdxCluster.getCrn()).getAccountId())) {
```

Comment:
should we say or multiazentitlementGranted()

Reply:
same here